source/common/config/grpc_subscription_impl.cc

```diff
@@ -121,5 +150,12 @@ void GrpcSubscriptionImpl::disableInitFetchTimeoutTimer() {
   }
 }
 
+// Returns the threshold used to trigger logging of long config updates.
+// TODO(adamsjob): make the threshold configurable, either through runtime or bootstrap
+std::chrono::milliseconds& GrpcSubscriptionImpl::updateDurationLogThreshold() {
+  static std::chrono::milliseconds* threshold_ms = new std::chrono::milliseconds(50);
+  return *threshold_ms;
+}
```

Comment:
std::chrono::milliseconds has a constexpr constructor, so this could be just a `constexpr` constant.

Reply:
Done, lmk if I need to make the name more globally unique.